src/utils/node/blog.js

```diff
@@ -0,0 +1,151 @@
+const path = require('path')
```

Comment:
like the split, but looks like createPages are not utils?

Reply:
I agree, but I could not come up with a better place for these helpers. I can move them to root `helpers` for example. Will it be better?

Reply:
Moved these helpers to `src/gatsby`